src/exit/tasks.py

```diff
@@ -0,0 +1,184 @@
+            block_number=block_number,
+        )
+
+
+def handel_exit_position(block_number: BlockNumber) -> None:
+    """Process osTokenExitRequests from graph and claim exited assets."""
+    # force claim for exit positions
+    max_ltv_percent = ostoken_vault_escrow_contract.liq_threshold_percent()
+    max_ltv_percent = max_ltv_percent // WAD
+    exit_requests = graph_ostoken_exit_requests(max_ltv_percent, block_number=block_number)
+    exit_requests = [
+        exit_request for exit_request in exit_requests if exit_request.exit_request.can_be_claimed
+    ]
+
+    logger.info('Force assets claim for %d exit requests...', len(exit_requests))
+    vault_to_harvest_params: dict[ChecksumAddress, HarvestParams | None] = {}
+
+    for os_token_exit_request in exit_requests:
+        vault = os_token_exit_request.vault
+        harvest_params = vault_to_harvest_params.get(vault)
+        if not harvest_params:
+            harvest_params = graph_get_harvest_params(vault)
+            vault_to_harvest_params[vault] = harvest_params
+
+        logger.info(
+            'Claiming exited assets: vault=%s, user=%s...',
+            vault,
+            os_token_exit_request.owner,
+        )
+        tx_hash = claim_exited_assets(
+            vault=vault,
+            user=os_token_exit_request.owner,
+            exit_request=os_token_exit_request.exit_request,
+            harvest_params=harvest_params,
+            block_number=block_number,
+        )
+        if tx_hash:
+            logger.info(
+                'Successfully claimed exited assets: vault=%s, user=%s...',
+                vault,
+                os_token_exit_request.owner,
+            )
+
+
+def handle_leverage_position(
+    position: LeveragePosition, harvest_params: HarvestParams | None, block_number: BlockNumber
+) -> None:
+    """
+    Submit force exit for leverage postion.
+    Also check for position active exit request and claim assets is possible.
+    """
+    if not can_force_enter_exit_queue(
+        vault=position.vault,
+        user=position.user,
+        harvest_params=harvest_params,
+        block_number=block_number,
+    ):
+        return
+
+    # claim active exit request
+    if position.exit_request and position.exit_request.can_be_claimed:
+        logger.info(
+            'Claiming exited assets for leverage positions: vault=%s, user=%s...',
+            position.vault,
+            position.user,
+        )
+        tx_hash = claim_exited_assets(
+            vault=position.vault,
+            user=position.user,
+            exit_request=position.exit_request,
+            harvest_params=harvest_params,
+            block_number=block_number,
+        )
+        if tx_hash:
+            logger.info(
+                'Successfully claimed exited assets for leverage positions: vault=%s, user=%s...',
+                position.vault,
+                position.user,
+            )
+        else:
+            return
+    logger.info(
```

Comment:
Lets check `can_force_enter_exit_queue` before force exiting as the position state has changed after claiming assets